Add/update Harvester landing page (#940)
rancher/rancher-docs: versioned_docs/version-2.7/integrations-in-rancher/harvester/overview.md

---
title: Overview
---

> Introduced in Rancher v2.6.1, [Harvester](https://docs.harvesterhci.io/) is an open-source hyper-converged infrastructure (HCI) software built on Kubernetes. Harvester installs on bare metal servers and provides integrated virtualization and distributed storage capabilities. Although Harvester operates using Kubernetes, it does not require users to know Kubernetes concepts, making it a more user-friendly application.

Reviewer: ```suggestion
Introduced in Rancher v2.6.1, Harvester is an open-source, hyper-converged infrastructure (HCI) software built on Kubernetes. Harvester installs on bare metal servers and provides integrated virtualization and distributed storage capabilities. Although Harvester operates using Kubernetes, it does not require knowledge of Kubernetes concepts, making it more user-friendly.
```
